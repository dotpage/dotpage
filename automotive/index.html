<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>automotive</title>
    <link rel="stylesheet" href="./css/styles.css">
</head>

<body>
    <header>
        <div class="wrapper">
            <nav>
                <img class="logo" src="./img/logo-white.png"></img>
                <ul class="menu">
                    <li>
                        <a href="">home</a>
                    </li>
                    <li>
                        <a href="#">about</a>
                    </li>
                    <li>
                        <a href="#">services</a>
                    </li>
                    <li>
                        <a href="#">gallery</a>
                    </li>
                    <li>
                        <a href="#">contact</a>
                    </li>
                </ul>
            </nav>
            <div class="row_header">
                <div class="main_title">
                    <h1>
                        <span>sell your</span> car now</h1>
                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
                        standard dummy text ever since the 1500s.</p>
                </div>
                <div class="form">
                    <h2>request a quote</h2>
                    <form action="#">
                        <input class="input_form" type="text">
                        <input class="input_form" type="email">
                        <input class="input_form" type="text">
                        <input class="input_form" type="text">
                        <div class="checkbox">
                            <input class="input_checkbox" type="checkbox">
                            <p>I accept the terms & conditions</p>
                        </div>
                        <div class="form_buttom">
                            <button type="submit">send</button>
                        </div>
                    </form>
                </div>
            </div>
        </div>
    </header>
    <section class="services">
        <div class="wrapper">
            <h2>
                <span>our</span> services</h2>
            <div class="block">
                <div class="price">
                    <img src="./img/heart.jpg" alt="">
                    <div class="item">Making your trip beautiful and easier</div>
                </div>
                <div class="price">
                    <img src="./img/tools.jpg" alt="">
                    <div class="item">Tools to help you succeed tommorow</div>
                </div>
                <div class="price">
                    <img src="./img/lamp.jpg" alt="">
                    <div class="item">Ideas that will blow you out the blue.</div>
                </div>
                <div class="price">
                    <img src="./img/compas.jpg" alt="">
                    <div class="item">Navigate your path to beautiful world.</div>
                </div>
            </div>
        </div>
    </section>
    <section class="our_store">
        <div class="wrapper">
            <div class="our_store_block">
                <p>We are in business for over 6 years providing amazing services to client and people love them to the core.
                    View our story to know more.</p>
                <div class="buttom">
                    <button type="button">our store</button>
                </div>
            </div>
        </div>
    </section>
    <section class="pricing">
        <div class="wrapper">
            <h2>pricing</h2>
            <div class="block">
                <div class="item">
                    <h3>starter</h3>
                    <div class="price">free</div>
                    <div class="text">
                        <p>free service</p>
                        <p>multiple accounts</p>
                        <p> management no</p>
                    </div>
                    <div class="button">
                        <button>try</button>
                    </div>
                </div>
                <div class="item popular">
                    <div class="tringle"></div>
                    <div class="incline"></div>
                    <h3>business</h3>
                    <div class="price">$97</div>
                    <div class="text">
                        <p>Free Service</p>
                        <p>Multiple Accounts</p>
                        <p> Management No</p>
                        <p>Unlimited Data</p>
                    </div>
                    <div class="button">
                        <button>try</button>
                    </div>
                </div>
                <div class="item">
                    <h3>professional</h3>
                    <div class="price">$297</div>
                    <div class="text">
                        <p>Free Service</p>
                        <p>Multiple Accounts</p>
                        <p> Management No</p>
                        <p>Unlimited Data</p>
                        <p>Whatever You Need</p>
                    </div>
                    <div class="button">
                        <button>try</button>
                    </div>
                </div>
            </div>

        </div>
    </section>
    <section class="testimonial_gallery">
        <div class="wrapper">
            <div class="block">
                <div class="testimonial_block">
                    <h2>testimonial</h2>
                    <div class="testimonial_items">
                        <div class="item">
                            <div class="text">
                                <p class="title">I have been involved with this comapny for ages and just want to say this is a great work
                                    by designcoon.</p>
                                <p>DJ Bravo - Frequent Travelera</p>
                            </div>
                            <img src="./img/man.png" alt="">
                        </div>
                        <div class="item">
                            <div class="text">
                                <p class="title">I have been involved with this comapny for ages and just .</p>
                                <p>DJ Bravo - Frequent Travelera</p>
                            </div>
                            <img src="./img/woman.png" alt="">
                        </div>
                    </div>
                </div>
                <div class="gallery_block">
                    <h2>gallery</h2>
                    <div class="gallery_items">
                        <img src="./img/woman-in-red.png" alt="">
                        <img src="./img/blueberry.jpg" alt="">
                        <img src="./img/lemon.jpg" alt="">
                        <img src="./img/black-and-white.jpg" alt="">
                        <img src="./img/legs.jpg" alt="">
                        <img src="./img/stones.jpg" alt="">
                        <a href="#">view more pictures</a>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <footer>
        <div class="wrapper">
            <div class="footer_block">
                    <p>All Rights Reserved | 2016 | Designed with love by <span>yaji2k</span></p>
                    <img src="./img/logo-red.png" alt="">
            </div>
        </div>
    </footer>
</body>

</html>