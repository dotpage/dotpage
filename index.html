<!DOCTYPE html>
<html lang="ru">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>dotepage</title>
    <link rel="stylesheet" href="./css/styles.css">
</head>

<body class="body">
    <div class="background-fixed"></div>
    <header class="header">
        <div class="wrapper">
            <div class="header-top">
                <div class="logo header-top__logo">
                    <h1 class="logo__title">.page
                        <span class="braces">{&#8230;}</span>
                    </h1>
                </div>

                <nav class="nav header-top__nav">
                    <ul class="nav__ul nav__theme_light">
                        <li class="nav__li nav_font-size_m">
                            <a class="nav__link" href="#portfolio">портфолио</a>
                        </li>
                        <li class="nav__li nav_font-size_m">
                            <a class="nav__link" href="#contacts">контакты</a>
                        </li>
                    </ul>
                </nav>
            </div>

            <div class="main-title">
                <div class="main-title__slider">
                    <div class="main-title__auto-height">
                        <div class="rectangle-slider frame-grey">
                            <img class="rectangle-slider__img" src="./img/golden.jpg" alt="Golden">
                            <img class="rectangle-slider__img" src="./img/automotive.jpg" alt="Automotive">
                        </div>
                    </div>
                </div>
                <div class="main-title__text">
                    <p class="main-title__text_bold">Здравствуйте!</p>
                    <p>Меня зовут Тимур Филиппов и я приветствую вас на моем сайте портфолио!</p>
                    <p> На сайте вы можете ознакомиться с моим портфолио, а так же найти контакты для связи со мной.</p>
                </div>
            </div>
    </header>
    <section id="portfolio" class="portfolio">
        <div class="wrapper">
            <h3 class="title portfolio-title">портфолио</h3>
            <div class="portfolio__items">
                <div class="portfolio__item frame-grey">
                    <a href="./golden/" target="_blank" class="portfolio__link">
                        <img src="./img/golden.jpg" alt="golden">
                    </a>
                </div>
                <div class="portfolio__item frame-grey">
                    <a href="./automotive/" target="_blank" class="portfolio__link">
                        <img src="./img/automotive.jpg" alt="automotive">
                    </a>
                </div>
            </div>
        </div>
    </section>
    <footer class="footer">
        <div class="wrapper footer__flex">
            <section id="contacts" class="contacts">
                <p class="title contacts-title">Контакты</p>
                <p class="email ">Email: <a class="email__link" href="mailto:yaji2k@gmail.com" title="написать на почту">yaji2k@gmail.com</a></p>
            </section>
            <nav class="nav footer__nav">
                    <ul class="nav__ul">
                        <li class="nav__li nav_font-size_m">
                            <a class="nav__link" href="#portfolio">портфолио</a>
                        </li>
                        <li class="nav__li nav_font-size_m">
                            <a class="nav__link" href="#contacts">контакты</a>
                        </li>
                    </ul>
                </nav>
        </div>
    </footer>
    <div class="btn-up"><img src="./img/up.svg" alt=""></div>
    <script src="./js/jquery-3.3.1.js"></script>
    <script src="./js/scripts.js"></script>
</body>

</html>