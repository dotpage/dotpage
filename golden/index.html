<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Document</title>
    <link rel="stylesheet" href="./css/main.css">
</head>

<body>
    <header>
        <div class="container clearfix">
            <nav>
                <img src="./img/logo.png" alt="logo">
                <ul>
                    <li><a href="#">home</a></li>
                    <li><a href="#">services</a></li>
                    <li><a href="#">portfolio</a></li>
                    <li><a href="#">about</a></li>
                    <li><a href="#">contact</a></li>
                </ul>
            </nav>
            <h2>Welcome To Our Studio!</h2>
            <h1>It’s nice to Meet you</h1>
            <div>
                <a href="#" class="button">Tell Me More</a>
            </div>
        </div>
    </header>
    <section>
        <div class="container clearfix">
            <div class="title">
                <h2>Services</h2>
                <p>Proin iaculis purus consequat sem cure</p>
            </div>
            <div>
                <div class="service">
                    <img src="./img/icon1.png" alt="">
                    <h3>
                        E-Commerce
                    </h3>
                    <p>
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
                <div class="service">
                    <img src="./img/icon2.png" alt="">
                    <h3>
                        Responsive Web
                    </h3>
                    <p>
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
                <div class="service">
                    <img src="./img/icon3.png" alt="">
                    <h3>
                        Web Security
                    </h3>
                    <p>
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
            </div>
        </div>
    </section>
    <section id="portfolio">
        <div class="container clearfix">
            <div class="title">
                <h2>Our portfolio</h2>
                <p>Proin iaculis purus consequat sem cure</p>
            </div>
            <div class="works">
                <img src="./img/img1.jpg" alt="">
                <img src="./img/img2.jpg" alt="">
                <img src="./img/img3.jpg" alt="">
            </div>
        </div>
    </section>
    <section>
        <div class="container clearfix">
            <div class="title">
                <h2>About Us</h2>
                <p>Proin iaculis purus consequat sem cure</p>
            </div>
            <about>
                <div class="box">
                    <div class="box__text">
                        <h3>July 2010 </h3>
                        <h3>Our Humble Beginnings</h3>
                        <p>
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
                        </p>
                    </div>
                    <div><img src="./img/about1.jpg" alt=""></div>
                </div>
                <div class="box">
                    <div class="box__text">
                        <h3>January 2011</h3>
                        <h3> Facing Startup Battles</h3>
                        <p>
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
                        </p>
                    </div>
                    <div><img src="./img/about2.jpg" alt=""></div>
                </div>
                <div class="box">
                    <div class="box__text">
                        <h3>December 2012 </h3>
                        <h3>Enter The Dark Days</h3>
                        <p>
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
                        </p>
                    </div>
                    <div><img src="./img/about3.jpg" alt=""></div>
                </div>
                <div class="box">
                    <div class="box__text">
                        <h3>February 2014 </h3>
                        <h3>Our Triumph</h3>
                        <p>
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
                        </p>
                    </div>
                    <div><img src="./img/about4.jpg" alt=""></div>
                </div>
            </about>
        </div>
    </section>
    <footer>
        <p>© Copyright 2014 FreebiesXpress.com</p>
    </footer>
</body>

</html>